service-front/module/Application/view/application/pages/identity_check_questions.twig

```diff
@@ -0,0 +1,44 @@
+<div class="govuk-width-container">
+    <main class="govuk-main-wrapper">
+        <form method="post" target="">
+            {% for questionBlock in questions_data %}
+                <div class="govuk-form-group {% if questionBlock.number != question %}moj-hidden{% endif %}">
```

Comment:
Rather than relying on CSS, it would be neater to passthrough previous answers in `<input type="hidden" />` inputs